### DISCUSSION TITLE
Using `route()->current('...*')` multiple time issue

### DISCUSSION THREAD

**Discussion by valpuia** [tighten/ziggy#720] [Q&A]
### Ziggy version

v1.8.2

### Laravel version

v10.46.0

### Description

When using `route()->current()` with wildcard multiple times cannot set nav-link active class.

You can clearly see from this video:
[Screencast from 02-28-2024 12:22:20 PM.webm](https://github.com/tighten/ziggy/assets/28250303/cb597ef9-24c6-420d-973a-9180b1deaa7f)

### Ziggy call and context

```js
<NavLink href="/users" :active="route().current('users.*')">{{ $t('Users') }}</NavLink>
<NavLink href="/posts" :active="route().current('posts.*')">{{ $t('Posts') }}</NavLink>

// NavLink components

<script setup>
defineProps({
    active: Boolean,
});
</script>

<template>
    <Link class="text-black dark:text-gray-300 hover:underline" :class="{ 'font-bold underline': active }">
    <slot />
    </Link>
</template>
```

### Ziggy configuration

```js
-
```

### Route definition

```php
Route::resource('posts', PostController::class);
Route::resource('users', UserController::class)->except(['show', 'edit']);
```

**Comment by bakerkretzmar** [comment]
`route().current('users.*')` in your example will only be evaluated once, when the component mounts. If you want the current route to be updated when you navigate to another page, you'll have to put it in a `computed()` or manually update it on navigation events. If you're using Inertia you can do this by listening for Inertia's `navigate` event. See #613 too.
